### PR TITLE
update doc to match config default value with TiKV codebase. (#5669)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #5669 to release-4.0
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In docs-cn repo:
git pr https://github.com/pingcap/docs-cn/pull/5889
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/docs-cn.git pr/5889:release-4.0-6fc87f99777c
```

---

Signed-off-by: Zhenhan Gong <zhenhan.gong@gmail.com>

### What is changed, added or deleted? (Required)

Match TiKV docs's config default value with TiKV codebase.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `grpc-concurrency`
 
 + gRPC 工作线程的数量。
-+ 默认值：4
++ 默认值：5
```
Comment (sticnarf) [inline tikv-configuration-file.md:32]: 不改

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -144,24 +144,28 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `use-unified-pool`
 
 + 是否使用统一的读取线程池（在 [`readpool.unified`](#readpoolunified) 中配置）处理存储请求。该选项值为 false 时，使用单独的存储线程池。通过本节 (`readpool.storage`) 中的其余配置项配置单独的线程池。
+<<<<<<< HEAD
 + 默认值：false
+=======
++ 默认值：如果本节 (`readpool.storage`) 中没有其他配置，默认为 true。否则，为了升级兼容性，默认为 false，请根据需要更改 [`readpool.unified`](#readpoolunified) 中的配置后再启用该选项。
+>>>>>>> 6fc87f99... update doc to match config default value with TiKV codebase. (#5669)
```
Comment (sticnarf) [inline tikv-configuration-file.md:151]: 不改

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -267,7 +271,7 @@ Coprocessor 线程池中线程的栈大小，默认值：10，单位：KiB|MiB|G
 ### `reserve-space`
 
 + TiKV 启动时预占额外空间的临时文件大小。临时文件名为 `space_placeholder_file`，位于 `storage.data-dir` 目录下。TiKV 磁盘空间耗尽无法正常启动需要紧急干预时，可以删除该文件，并且将 `reserve-space` 设置为 `0MB`。
-+ 默认值：2GB
++ 默认值：5GB
```
Comment (sticnarf) [inline tikv-configuration-file.md:274]: 不改

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -561,7 +559,7 @@ raftstore 相关的配置项。
 ### `merge-check-tick-interval`
 
 + 触发 merge 完成检查的时间间隔。
-+ 默认值：10s
++ 默认值：2s
```
Comment (sticnarf) [inline tikv-configuration-file.md:562]: 暂不改，等 https://github.com/tikv/tikv/pull/9676

**Comment by TomShawn** [comment]
@sticnarf PTAL again, thanks!

**Review by sticnarf — APPROVED** [review]
LGTM

**Pull request merged**